packages/react-aria-components/src/Table.tsx

```diff
@@ -1175,14 +1197,21 @@ export interface CellRenderProps {
    * Whether the cell is currently hovered with a mouse.
    * @selector [data-hovered]
    */
-  isHovered: boolean
+  isHovered: boolean,
+  /**
+   * Whether the cell is read only.
+   * @selector [data-readonly]
+   */
+  isReadOnly: boolean
 }
 
 export interface CellProps extends RenderProps<CellRenderProps> {
-  /** The unique id of the cell. */
-  id?: Key,
```

Comment:
I removed the `id` because it wasn't being forwarded to the element anyways and it caused issues after adjusting the key structure. In case we want those back, some further debugging is required.

Reply:
What kind of issues did you run into? The `id` if provided is used by the CollectionBuilder as the key in the key map if provided, otherwise it autogenerates one (similar to Columns and Rows). Probably not super important for now but I imagine it will be if/when we support cell actions

Reply:
Seems to have sorted itself out with the latest commits so i added this back 👍